rlImGui.cpp

```diff
@@ -99,7 +102,13 @@ static void rlImGuiNewFrame(float deltaTime)
     }
     else 
     {
-        io.DisplayFramebufferScale = ImVec2(1.0f, 1.0f);
+
+#if !__APPLE__
+      io.DisplayFramebufferScale = ImVec2(1.0f, 1.0f);
+#else
+      float backingScale = getBackingScaleFactor();
+      io.DisplayFramebufferScale = ImVec2(backingScale, backingScale);
+#endif
```

Comment:
Also this code would only trigger if width or height are <= 0. What values are you getting for width and height that let you get in here?

Reply:
I discovered that using int width = int(io.DisplaySize.x), height = int(io.DisplaySize.y); correctly retrieves the non-Retina display size. However, after invoking glfwGetFramebufferSize(glfwGetCurrentContext(), &width, &height);, both the width and height are set to 0. I suspect this behavior came from Apple's OpenGL implementation. While I'm not entirely certain about the extent to address this issue especially since it seems to be a macOS-exclusive issue, I think GetWindowScaleDPI would solve the issue.